### ISSUE TITLE
errors spam in server-main log

### ISSUE THREAD

**Issue by cyraxan** [slightlycolorblind/Wildcraft#3]
3.2.2022 10:13:50 [Error] BETransient exiting at 512389, 110, 518052 cannot find block attributes for wildcraft:leafygroundvegetable-stingingnettle-harvested. Will stop transient timer
3.2.2022 10:14:44 [Error] BETransient exiting at 512555, 114, 517658 cannot find block attributes for wildcraft:leafygroundvegetable-dock-harvested. Will stop transient timer
3.2.2022 10:14:44 [Error] BETransient exiting at 512557, 114, 517657 cannot find block attributes for wildcraft:leafygroundvegetable-dock-harvested. Will stop transient timer
3.2.2022 10:14:44 [Error] BETransient exiting at 512575, 115, 517670 cannot find block attributes for wildcraft:leafygroundvegetable-stjohnswort-harvested. Will stop transient timer
3.2.2022 10:14:44 [Error] BETransient exiting at 512575, 114, 517716 cannot find block attributes for wildcraft:leafygroundvegetable-minerslettuce-harvested. Will stop transient timer
3.2.2022 10:14:44 [Error] BETransient exiting at 512575, 114, 517715 cannot find block attributes for wildcraft:leafygroundvegetable-minerslettuce-harvested. Will stop transient timer
3.2.2022 10:14:44 [Error] BETransient exiting at 512574, 114, 517716 cannot find block attributes for wildcraft:leafygroundvegetable-minerslettuce-harvested. Will stop transient timer
3.2.2022 10:14:44 [Error] BETransient exiting at 512534, 115, 517742 cannot find block attributes for wildcraft:leafygroundvegetable-rosemary-harvested. Will stop transient timer

Removing blocks and objects at given coords not helping.

**Comment by gabriella-campos-davis** [comment]
Interesting, BETransient isn't something I wrote. This could be related to me using so many of the same BlockEntity, I'll try to do some tests, hopefully can find a fix.

Did the plant at the given coords stop working? is there associated lag with the reports? are the reports happening consistently, or is it a one time thing?

**Comment by cyraxan** [comment]
I suspect that's legacy of early existent, but harvested or destroyed plants. Or probably tails of "? blocks" destroyed in period, when server was running without Wildcraft mod.
Server was generated on 1.16.0 version VS with wildcraft 1.1.0, and due to massive lags wildcraft mod was disabled on server, but generated blocks on populated chunks still produced massive error spam in server log.

**Comment by gabriella-campos-davis** [comment]
So, the error spam didn't happen until after wildcraft was disabled? Just to be clear, you installed the mod, generated the world, got a lot of lag (I'll need to work on that ), disabled the mod, and that's when the error spam happened?

**Comment by cyraxan** [comment]
I inspected logs to ensure correct details, some extracts:

world generation

```
17.1.2022 14:53:41 [Notification] Game Version: v1.16.0 (Stable)
17.1.2022 15:00:16 [Notification] External Origins in load order: modorigin@c:\VSS\assets\creative\, modorigin@c:\VSS\assets\survival\, mod@AdvancedSmithing1.1.0.zip, mod@AncientTools_V1.3.1.zip, mod@animalcages_v2.0.5.zip, mod@axleinblocks_v1.0.12.zip, mod@blacksmithname_v1.0.1.zip, mod@cats_v1.2.1_petai_v1.2.x.zip, mod@cavepaintings_v1.16.0-1.3.0.zip, mod@ExtraChestsCC_v1.5.0.zip, mod@fancydoors_v1.16.0-1.1.0.zip, mod@necessaries_v1.16.0-1.1.0.zip, mod@NeededThings.zip, mod@NightWatcher-v1.2.zip, mod@petai_v1.2.1.zip, mod@plantexpansion.zip, mod@PrimitiveSurvival2.8.0.zip, mod@qptech1.6.2.zip, mod@spyglass_v0.4.2.zip, mod@tinytweaks_v1.0.0.zip, mod@tradeomat_v1.16.0-1.6.0.zip, mod@ViesCraft-VS1.16.0-v2.0.0.zip, mod@weightmod_v1.0.0.zip, mod@wolftaming_v1.2.1_petai_v1.2.x.zip, mod@alchemy-1.5.4.zip, mod@Bonfires-0.2.zip, mod@CarryCapacity-VS1.16.0-v0.6.4.zip, mod@ExpandedLiquidContainers-v1.1.0.zip, mod@HelveHammerExtensions_v1.16.0-pre.8_v1.2.0.zip, mod@Lichen-v1.6-VS1.16.0.zip, mod@medievalexpansion-3.7.1.zip, mod@MoreBags-1.0.0.zip, mod@StoneBrickRebalance-v1.0.0.zip, mod@StoneQuarry_v1.16.0-rc.2_v1.5.0.zip, mod@TeleportationNetwork_v1.16.0-pre.8_v1.5.1.zip, mod@WoodBarrels-v1.0.0.zip, mod@workbench-expansion-1.4.1.zip, mod@xlib_v0.6.3.zip, mod@xmelee_0.0.9.zip, mod@xrowboat_0.0.2.zip, mod@AnvilMetalRecovery_V0.16-Release.zip, mod@Bullseye 2.2.4.zip, mod@DoffAndDonAgain_v2.0.0.zip, mod@wildcraft-v1.1.2.zip, mod@xskills_v0.6.5.zip, mod@AlphaWeaponPack_Patchv2.0.0.zip, mod@AnMiTech_v1.0.6.zip, mod@TemporalTinkerer_v2.3.0.zip
17.1.2022 15:07:03 [Notification]     Mod 'wildcraft-v1.1.2.zip' (wildcraft):
soon after harvesting or destroying some wildcraft crops
17.1.2022 17:12:06 [Error] BETransient exiting at 510926, 115, 512750 cannot find block attributes for wildcraft:leafygroundvegetable-bearleek-harvested. Will stop transient timer
```

Also errors suspected to wilcraft during this server session

```
18.1.2022 01:25:54 [Error] Exception: System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at weightmod.src.weightmod.OnPlayerNowPlaying(IServerPlayer byPlayer)
   at Vintagestory.API.Common.PlayerDelegate.Invoke(IServerPlayer byPlayer)
   at Vintagestory.Server.ServerEventManager.TriggerPlayerNowPlaying(IServerPlayer player)
18.1.2022 01:32:45 [Error] An error was thrown when generating chunk column X=16029,Z=16029 - it will probably not generate correctly!. Exception System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.ServerMods.GenCreatures.CanSpawnAt(IBlockAccessor blockAccessor, EntityProperties type, BlockPos pos, Vec3d posAsVec, BaseSpawnConditions sc, Single rain, Single temp, Single forestDensity, Single shrubsDensity)
   at Vintagestory.ServerMods.GenCreatures.TrySpawnGroupAt(BlockPos origin, Vec3d posAsVec, EntityProperties entityType, EntityProperties[] grouptypes)
   at Vintagestory.ServerMods.GenCreatures.OnChunkColumnGen(IServerChunk[] chunks, Int32 chunkX, Int32 chunkZ, ITreeAttribute chunkGenParams)
   at Vintagestory.Server.ServerSystemSupplyChunks.TryPopulateChunk(ChunkColumnGetRequest chunkRequest)
18.1.2022 05:28:11 [Error] BETransient exiting at 510926, 115, 512750 cannot find block attributes for wildcraft:leafygroundvegetable-bearleek-harvested. Will stop transient timer

Spam in logs like that to 24.01.2022 22:40 when server was restarted without wildcraft

24.1.2022 22:40:01 [Notification] Mods, sorted by dependency: advancedsmithing, weaponpackapatch, ancienttools, animalcages, tonwexp, axleinblocks, cats, necessaries, neededthings, nightwatcher, petai, PolServerUtilities, primitivesurvival, qptech, locustmod, tinytweaks, tradeomat, viesblocks, viescraft, game, wolftaming, alchemy, bonfires, carrycapacity, exlico, helvehammerext, lichen, medievalexpansion, morebags, playercorpse, stonequarry, tpnet, usefulstuff, creative, survival, workbenchexpansion, xmelee, xrowboat, metalrecovery, bullseye

```
And massive log spam during server start and client joins:

`24.1.2022 22:44:03 [Error] Failed loading a blockentity in a chunk. Will discard it. Sorry. Exception logged to verbose debug.`

Seems to happened every time as server tries to present wildcraft block info to new client
Same spam continuous in logs to 02.02.2022 when server was updated to 1.16.3 with wildcraft 1.1.4

After update server to 1.16.3 and wildcraft to 1.1.4 I got back spam like:

`9.2.2022 17:24:12 [Error] BETransient exiting at 511720, 111, 518101 cannot find block attributes for wildcraft:leafygroundvegetable-mint-normal. Will stop transient timer`

I tried to place in given coordinates blocks named wildcraft crops, or another crops, but it have no effect on error spam in logs.

**Comment by cyraxan** [comment]
`I tried to place in given coordinates blocks named wildcraft crops, or another crops, but it have no effect on error spam in logs.`

After careful inspection, this turned out not to be true.
I had already fixed almost all the defective blocks by putting the correct cultures on the exact blocks. 
But it took a huge amount of time to do it manually.
It would be nice to automate this somehow or fix it with mod tools.
